### PR TITLE
Delete option

### PR DESCRIPTION
**Pull request by priyushasunkara**

### CODE REVIEW

**Review by GcvKishore — APPROVED** [review]
Working as expected.

**Review by Amulya301 — APPROVED** [review]
Deleting exam and questions are working correctly.

File: exam_app/views.py [hunk 1]
```diff
@@ -23,15 +23,30 @@ def createExam(request):
...
+    exam = MakeExam.objects.all(id=exam_id)
+    question = MakeQuestion.objects.filter(exam_model=exam).delete()
+    questions_list = MakeQuestion.objects.filter(exam_model_id=exam_id)
+
+    return render(request, 'exam_app/edit-exam.html', {
+        'exam': exam,
+        'questions': questions_list
+    })
+
+
```
Comment (Vedvund) [inline exam_app/views.py]: If you are not using this function you can delete it...

File: exam_app/views.py [hunk 2]
```diff
@@ -79,6 +94,9 @@ def addQuestion(request, exam_id):
 
 
 def viewAllExamsInstructors(request):
+    if request.method == 'POST':
+        delete_exam = request.POST['delete_exam']
+        MakeExam.objects.get(id = delete_exam).delete()
```
Comment (Vedvund) [inline exam_app/views.py]: Here you are only deleting the exam objects and the questions related to the exam are still in the database. (In this case, you need to delete the related questions 1st)

Use this code before line 99
MakeQuestion.objects.filter(exam_model__id=delete_exam).delete()

**Review by Vedvund — CHANGES_REQUESTED** [review]
Added comments

**Review by Vedvund — APPROVED** [review]

**Pull request merged**